kolibri/plugins/coach/assets/src/views/quizzes/ExamsRootPage.vue

```diff
@@ -324,7 +324,6 @@
         showOpenConfirmationModal,
         showCloseConfirmationModal,
         activeQuiz,
-        learnOnlyDevicesExist,
```

Comment:
@marcellamaki I was suspicious of this not being required, as it feels like linking to the list of users' sync statuses has got lost in the refactor? Might need a follow up issue for a regression.

Reply:
Yes, it does seem like if that's not required, we've lost some functionality.... I'll investigate and open an issue

Reply:
Thank you!

Reply:
we did lose some functionality, but in a different way than we thought! [follow up issue here](https://github.com/learningequality/kolibri/issues/12947)